### PR TITLE
changed 'wake_up' word to use vosk instead of pocketsphinx

### PR DESCRIPTION
**Pull request by builderjer**
This was suggested in [chats here](https://matrix.to/#/!EOygeDJPfJQOfNacqH:matrix.org/$9w7ZNfXMg8Ke4k-y30OjLQqpmtvHKg_I3iNDqC4SbpU?via=matrix.org&via=friends.famedly.de&via=kde.org), so I thought I would to a PR and see what you thought.

### CODE REVIEW

**Comment by j1nx** [comment]
Tested this on a running device;

2022-11-15 08:36:44.398 - voice - mycroft.messagebus.load_config:load_message_bus_config:33 - INFO - Loading message bus configs
2022-11-15 08:36:46.695 - voice - mycroft.util.process_utils:start_message_bus_client:135 - INFO - Connected to messagebus
2022-11-15 08:36:46.784 - voice - mycroft.listener:create_hotword_engines:333 - INFO - creating hotword engines
2022-11-15 08:36:46.791 - voice - ovos_plugin_manager.wakewords:load_module:85 - INFO - Loading "hey_mycroft" wake word via ovos-precise-lite
2022-11-15 08:36:47.595 - voice - ovos_plugin_manager.wakewords:load_module:92 - INFO - Loaded the Wake Word plugin ovos-precise-lite
2022-11-15 08:36:47.606 - voice - ovos_plugin_manager.wakewords:load_module:85 - INFO - Loading "wake_up" wake word via ovos-ww-plugin-vosk
2022-11-15 08:36:47.724 - voice - ovos_plugin_manager.wakewords:load_module:92 - INFO - Loaded the Wake Word plugin ovos-ww-plugin-vosk
2022-11-15 08:36:49.278 - voice - mycroft.listener.mic:__init__:362 - INFO - VAD method: vad_and_ratio
2022-11-15 08:36:49.280 - voice - mycroft.listener.mic:__init__:368 - INFO - Creating VAD engine: ovos-vad-plugin-webrtcvad
2022-11-15 08:36:49.353 - voice - mycroft.listener.stt:create:12 - INFO - Creating STT engine: ovos-stt-plugin-selene
2022-11-15 08:36:53.626 - voice - mycroft.listener.mic:adjust_for_ambient_noise:982 - INFO - Silence threshold adjusted to 307.4
2022-11-15 08:36:53.929 - voice - mycroft.listener.mic:adjust_for_ambient_noise:982 - INFO - Silence threshold adjusted to 418.0

Bare in mind the vosk-ww plugin needs to be installed so most likely be changed within the right requirements file as well. Will do some testing over the coming days and report back.

File: mycroft/configuration/mycroft.conf [hunk 1]
```diff
@@ -470,10 +470,11 @@
     },
 
     "wake_up": {
-        "module": "ovos-ww-plugin-pocketsphinx",
-        "phonemes": "W EY K . AH P",
-        "threshold": 1e-20,
-        "lang": "en-us",
+        "module": "ovos-ww-plugin-vosk",
```
Comment (JarbasAl) [inline mycroft/configuration/mycroft.conf:473]: needs to be added to requirements.txt and stt extra requirements

Comment (builderjer) [reply mycroft/configuration/mycroft.conf:473]: I think I put the appropriate versions

File: requirements/requirements.txt [hunk 2]
```diff
@@ -18,6 +18,7 @@ ovos-tts-plugin-mimic2~=0.1, >=0.1.5
 ovos-tts-plugin-google-tx~=0.0, >=0.0.3
 ovos-ww-plugin-pocketsphinx~=0.1
 ovos-ww-plugin-precise~=0.1
+ovos-ww-plugin-vosk~=0.1
```
Comment (JarbasAl) [inline requirements/requirements.txt:21]: specify minimum version also, tests are failing this should fix them `ovos-ww-plugin-vosk~=0.1, >=0.1.1a1`

**Pull request closed without merging**